ReactQt/runtime/src/bridge.cpp

```diff
@@ -581,7 +583,7 @@ void Bridge::invokeModuleMethod(int moduleId, int methodId, QList<QVariant> args
     }
 
     // readable log of methods invoked via bridge
-    // qDebug() << "INVOKE: " << moduleData->name() << "::" << method->name() << "( " << args << " )";
+    //    qDebug() << "INVOKE: " << moduleData->name() << "::" << method->name() << "( " << args << " )";
```

Comment:
Do you mean whitespaces or the commented code itself?

I'd prefer to leave that commented line in the code because it is extremely helpful. I'm uncommenting it almost every time during development :)

Reply:
Yes, only whitespaces :)